src/MembraneWebRTC.types.ts

```diff
@@ -321,15 +321,15 @@ export type EndpointsUpdateEvent<
   >[];
 };
 
-export type IsCameraOnEvent = boolean;
+export type IsCameraOnEvent = { [key: string]: boolean };
```

Comment:
You can be more specific here: https://www.typescriptlang.org/play?#code/FDAuE8AcFMAIEkDOBhAhgW2gJ1QeQHYCiAbtPqLALywDesA2gNbTgBcsioWAlvgOYBddgCMA9qIA20VPlgBfANxgocJGkw4CJMqABCAVz3RQobFVqwARGozY8+SyPFSZ8pcADGo-J1ipzNJaoiAAmAGYhiBFhjrBc+tCKnt6+wgHWKLaaDuzxie7+wQiZGvba5EppRTalWqQVwAD0jbAAAqCIALTQAB4wHqDdWFiiWMCFiMXqdnU6BkYm2JV+kzUzRPV6hrrGplgKQA